src/main/java/mat/client/export/measure/ManageMeasureExportView.java

```diff
@@ -8,10 +8,7 @@
 import com.google.gwt.user.client.ui.Widget;
 import mat.client.buttons.CancelButton;
 import mat.client.buttons.SaveButton;
-import mat.client.shared.ErrorMessageAlert;
-import mat.client.shared.MatContext;
-import mat.client.shared.MessageAlert;
-import mat.client.shared.SpacerWidget;
+import mat.client.shared.*;
```

Comment:
Just curious, is it preferred to import * instead of individual ones?

Reply:
I have my Intelli J settings set to max 5, I guess it did update automatically. I modified it @sb-cecilialiu